Source/WebCore/Modules/webauthn/fido/DeviceRequestConverter.cpp

```diff
@@ -220,6 +221,33 @@ Vector<uint8_t> encodeGetAssertionRequestAsCBOR(const Vector<uint8_t>& hash, con
     return cborRequest;
 }
 
+Vector<uint8_t> encodeBogusRequestForAuthenticatorSelection()
+{
+    CBORValue::MapValue cborMap;
+    cborMap[CBORValue(1)] = CBORValue(Vector<uint8_t> { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 });
```

Comment:
Nit: I wonder if this could just be something like:
`CBORValue(Vector<uint8_t>(0, 32));`